File: Bunshack_Restaurant.Server/Models/Order.cs
namespace Bunshack_Restaurant.Server.Models
{
    public class Order
    {
        public Guid Id { get; set; }
        public string? CustomerName { get; set; }
        public DateTime OrderDate { get; set; }
        public ICollection<OrderMenu> OrderMenus { get; set; } = new List<OrderMenu>();
        public string? UserId { get; set; }
        public User? User { get; set; }
    }
}

File: bunshack_restaurant.client/src/types/Order.ts
import { User } from './User';

export interface Order {
    id: string;
    customerName: string | null;
    orderDate: Date;
    orderMenus: OrderMenu[];
    userId: string | null;
    user: User | null;
    totalPrice: number;
}

export interface OrderMenu {
    orderId: string;
    menuId: string; 
    quantity: number;
}
